webpack.config.js

```diff
@@ -1,14 +1,25 @@
-const path = require('path');
-const outputDir = path.join(__dirname, "build/");
+const path = require("path");
 
-const isProd = process.env.NODE_ENV === 'production';
+const isProd = process.env.NODE_ENV === "production";
```

Comment:
Nit – we are _never_ using `webpack` for prod at this point, so no need to check for this flag.